src/components/SchemaViewer.svelte

```diff
@@ -56,21 +57,28 @@
 </script>
 
 <style>
-  .schema-browser {
-    @apply p-2;
-    @apply border-4;
-    @apply border-gray-400;
-    max-height: 400px;
-    overflow: scroll;
+  .schema-viewer {
+    margin-top: 2rem;
+    .schema-browser {
```

Comment:
Interesting, I didn't know it was possible to nest css declarations like this!

Reply:
You can't do it with css, this is actually sass syntax: https://sass-lang.com/documentation/style-rules ;)

Reply:
Nice!